DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj

```diff
@@ -18,9 +18,18 @@
     <Description>
       Automation support for CompUI
     </Description>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <PlatformTarget>x64</PlatformTarget>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="..\stylecop.json" Link="stylecop.json" />
+  </ItemGroup>
+  
   <ItemGroup>
```

Comment:
@jbarnes16503 Is there a way to align these <PackageReference> (include; version) details set within DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj  to those <dependency> (id; version) details within DFC.TestAutomation.UI.nuspec. i.e. Should they always match? There are additional style cop references here.

Reply:
Style Cop needs to be referenced in the .csproj file in order to highlight styling violations throughout the project. You wouldn't expect to see this reference in the .nuspec. To address your alignment query, we have a ticket in the backlog to remove the .nuspec entirely and go with the .csproj to package the NuGet. This has a few technical challenges, hence why it's in the backlog but this will eliminate the need to align the two files. At eh moment the .csproj isnt used to package up the NuGet.